app/models/think_feel_do_dashboard/concerns/group.rb

```diff
@@ -57,7 +57,7 @@ def create_moderating_participant
       end
 
       def create_participant(study_id)
-        password = SecureRandom.hex(64)
+        password = SecureRandom.hex(32) + SecureRandom.hex(32).upcase
```

Comment:
Good question.  `SecureRandom.hex(32)` only returns lowercase characters and whole numbers.  The password changes that are now require need at least one uppercase letter. :smile: